Sources/SwifterSwift/SwiftStdlib/ArrayExtensions.swift

```diff
@@ -18,32 +15,74 @@ public extension Array {
     ///        ["e", "l", "l", "o"].prepend("h") -> ["h", "e", "l", "l", "o"]
     ///
     /// - Parameter newElement: element to insert.
-    public mutating func prepend(_ newElement: Element) {
+    mutating func prepend(_ newElement: Element) {
         insert(newElement, at: 0)
     }
 
-    /// SwifterSwift: Safely Swap values at index positions.
+    /// SwifterSwift: Safely swap values at given index positions.
     ///
     ///        [1, 2, 3, 4, 5].safeSwap(from: 3, to: 0) -> [4, 2, 3, 1, 5]
     ///        ["h", "e", "l", "l", "o"].safeSwap(from: 1, to: 0) -> ["e", "h", "l", "l", "o"]
     ///
     /// - Parameters:
     ///   - index: index of first element.
     ///   - otherIndex: index of other element.
-    public mutating func safeSwap(from index: Index, to otherIndex: Index) {
+    mutating func safeSwap(from index: Index, to otherIndex: Index) {
         guard index != otherIndex else { return }
         guard startIndex..<endIndex ~= index else { return }
         guard startIndex..<endIndex ~= otherIndex else { return }
         swapAt(index, otherIndex)
     }
 
-    /// SwifterSwift: Separates an array into 2 arrays based on a predicate.
+    /// SwifterSwift: Keep elements of array while condition is true.
```

Comment:
How this got added back? I remember this being moved to Collection.

Reply:
It could be that I added it by accident while updating from `master` 😅